coralproject/talk
# [CORL-572] Filter rejected comments from reply counts on user comment history

## What does this PR do?

Update the replyCount resolver on comments to filter out rejected and pre-mod status replies.

## How do I test this PR?

- Create a comment using a commenter account
- Have another user reply at least twice
- Have a moderator account reject one of the replying comments
- Head back to the original, root commenter account, check their My Profile > My Comments > Comment History
- Reply count under the comment should not include the rejected comment
